test/test_grade.py

```diff
@@ -124,12 +162,10 @@ def test_notebooks_with_pdfs(self):
         )
         self.assertEqual(sorted(dir1_contents), sorted(dir2_contents), f"'{TEST_FILES_PATH}notebooks/' and 'test/submission_pdfs' have different contents")
 
-
-
+    def tearDown(self) -> None:
         # remove the extra output
-        cleanup_command = ["rm", "-rf", "test/final_grades.csv", "test/submission_pdfs", "test/final_grades.csv", TEST_FILES_PATH + "autograder.zip"]
+        cleanup_command = ["rm", "-rf", "test/final_grades.csv", "test/submission_pdfs", "test/final_grades.csv"]
         cleanup = subprocess.run(cleanup_command, stdout=PIPE, stderr=PIPE)
-        self.assertEqual(len(cleanup.stderr), 0, cleanup.stderr.decode("utf-8"))
```

Comment:
How come you removed this line? I think with `PIPE` the cleanup process won't reveal any errors if we don't perform this check.

Reply:
Well that was a mistake.